docs/beamrunner/release/1.2.0/io.md

```diff
@@ -14,89 +14,75 @@ next:
+1. Click **Object Storage**.
+1. Change the Service name to something meaningful to you or leave the default name.
+1. For **Pricing Plan**, click **Lite**.
+1. Click **Create**. IBM Cloud will return you to the Dashboard while the service is provisioned.
+1. Once provisioned, click on your object storage service in the Dashboard to open the **Buckets** page for the service.
+1. Click **Create Bucket**.
+1. Provide a unique bucket name (e.g., username-beam-bucket), resiliency, location, and storage class.
+1. Click **Create**
 
 ### Setting up credentials for the service
 
-To use the storage from Beam applications, you must specify the IBM Cloud service credentials. You can specify the credentials by setting environment variables or by using Swift command-line options. For more information about the Swift command-line options, see [Configuring the CLI to use Swift and Cloud Foundry commands](https://console.bluemix.net/docs/services/ObjectStorage/os_configuring.html).
-
-1. After the service is provisioned, select the object storage service that you created from the dashboard to open the **Manage** page for the service.
-2. On the service page, click **Service credentials**.
-3. If necessary, create a credential by clicking **New credential**. Use the default information and click **Add**.
-4. Click **View credentials**.
-5. On the computer where Streams Runner is installed, create the following environment variables from the fields that are shown in the credentials.  
-
-| Environment variable | Command-line option           | Credentials field | Environment variable example                                                 |
-|----------------------|------------------|-------------------|---------------------------------------------------------|
-| `OS_USER_ID`           | `--swiftUserId`    | `userId`            | `export OS_USER_ID='2b670d77432e4cf2bd128ef9ff61fa56'`    |
-| `OS_PASSWORD`          | `--swiftPassword`  | `password`          | `export OS_PASSWORD='f1H/~BIO.=s0wuT9'`                  |
-| `OS_PROJECT_ID`        | `--swiftProjectId` | `projectId`         | `export OS_PROJECT_ID='80301e24254f4ffb81d53f0cddccad78'` |
-| `OS_REGION_NAME`       | `--swiftRegion`    | `region`            | `export OS_REGION='dallas'`                               |
+To use the storage service from Beam applications, you must specify the IBM Cloud service credentials. Since the `s3` FileSystem is based on AWS object storage, you will need to create credentials with HMAC keys.
 
-**Tip**: For MacOS, the Swift command of OpenStack might collide with the existing Xcode Swift command. To avoid the conflicts, create a Python virtual environment, and install the Swift client in the virtual environment.
+1. On the service page, click **Service credentials**.
+1. Create a credential by clicking **New credential**.
+1. Put `{"HMAC":true}` in the **Add Inline Configuration Parameters (Optional) field** and click **Add** to create the new credentials.

```

Comment:
Question: what is "HMAC"?

Reply:
Unfortunately the docs do not mention it, but I found "Hash Message Authentication Code" that seems to fit